modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -507,7 +508,12 @@ private CompletableFuture<?> processRequest(ReplicaRequest request, @Nullable Bo
      * @param request Tx recovery request.
      * @return The future is complete when the transaction state is finalized.
      */
-    private CompletableFuture<Void> processTxRecoveryAction(TxRecoveryMessage request) {
+    private CompletableFuture<Void> processTxRecoveryAction(TxRecoveryMessage request, String senderId) {
```

Comment:
could you please create a separate method like `private CompletableFuture<TransactionMeta> triggerTxRecovery(UUID txId)` for this, because it is needed to trigger the recovery from tx state resolution process.